src/SfxWeb/src/app/Models/RawDataTypes.ts

```diff
@@ -1145,4 +1149,24 @@ export interface IRawApplicationNameInfo{
 export interface IRawServiceNameInfo{
         Id: string;
         Name: string;
-    }
+}
+    
+export interface IRawApplicationMetadata{
+    ArmMetadata: IRawArmMetadata;
+    HealthState: string;
```

Comment:
I believe so, but this was 8 months ago, and I could no longer find where I discovered these properties. Since we only expose properties that are used and these are not currently being used, we can delete them.